terminology.md

```diff
@@ -53,6 +53,11 @@ The Resource Description Framework (RDF) of the Semantic Web [rdf11-concepts](ht
 ## WoT Server
 An entity that exposes a network interface consistent with a WoT Thing Description. WoT Server is also used to refer to a Servient in server role only.
 
+## WoT API
+A network interface consistent with a WoT Thing Description exposed by a WoT Server.
+
```

Comment:
"network interface" has hardware connotations. 
https://en.wikipedia.org/wiki/Network_interface
We need to express the layer WoT is operating on.
We used to have a definition for a "WoT interface" in the IG Best Practices doc. 
http://w3c.github.io/wot/current-practices/wot-practices.html#wot-interface
We'd need to compact that (not very concise) definition.
(thinking on a proposal)

Reply:
What about 

<dt><dfn>WoT Interface</dfn></dt>
<dd>
  Resource-oriented, REST-ful Web interface that allows access to <a>Things</a> exposed over a network using different <a>Protocol Bindings</a>, as defined by <a>Thing Description</a>s.
</dd>

Reply:
To me it feels that "REST-ful" is -- yet again -- just used for its buzzwordiness.

The important part was to distinguish "WoT API" from "Scripting API" as many people through those two together. Also some of the Protocol Bindings will not be Web, so it is not a Web interface per se.

---

#### WoT Interface
The network-facing interface of a Thing as defined by its Thing Description. WoT Interfaces are a superset of Web Interfaces, as Protocol Bindings can also include non-Web protocols such as MQTT or ZigBee. While sometimes called "WoT API", it is not to be confused with the Scripting API, which is interfaces with application scripts within a Servient.

Reply:
I just notieced, that this requires a tweak to "Interaction Model" and "Interactions", as they describe both, the network-facing interface (WoT Interface) and the application-facing interface (Scripting API).

Reply:
Interaction Model definition updated (see https://github.com/w3c/wot-architecture/blob/master/terminology.md)

Reply:
> WoT Interface
The network-facing interface of a Thing as defined by its Thing Description. 

This.

Reply:
Meaning taking just the first sentence and move the rest to somewhere in the Architecture document?

Reply:
Yes, I think the first part is concise enough.

Reply:
This is closed now, but the text is not changed.

Reply:
https://github.com/w3c/wot-architecture/commit/67ffd835380d53b86260f0240afedda07e7de251

I committed it directly. See entry below RDF in https://github.com/w3c/wot-architecture/blob/master/terminology.md

Or what do you mean?